resources/views/pages/leaderboard/[leaderboard]/comments.blade.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use App\Models\Leaderboard;
+use Illuminate\View\View;
+
+use function Laravel\Folio\{middleware, name, render};
+
+middleware(['auth', 'can:viewAny,' . Leaderboard::class]);
```

Comment:
```suggestion
middleware(['can:viewAny,' . Leaderboard::class]);
```

Probably should yank off this auth middleware.

Reply:
done.